src/unix/notbsd/linux/other/mod.rs

```diff
@@ -548,6 +548,13 @@ pub const NLA_F_NESTED: ::c_int = 1 << 15;
 pub const NLA_F_NET_BYTEORDER: ::c_int = 1 << 14;
 pub const NLA_TYPE_MASK: ::c_int = !(NLA_F_NESTED | NLA_F_NET_BYTEORDER);
 
+#[repr(C)]
+pub enum idtype_t {
+    P_ALL,
+    P_PID,
+    P_PGID
+}
```

Comment:
I lifted the the constant definition to `linux/mod.rs` from `/linux/other/mod.rs` because `sys/wait.h` _should_ be standard cross all GNU/Linux platforms. 

The type+explicitly declared constants is added. Waiting for Travis-ci to report.